src/ripple/basics/TaggedCache.h

```diff
@@ -72,7 +72,7 @@ class TaggedCache
 public:
     // VFALCO TODO Change expiration_seconds to clock_type::duration
     TaggedCache (std::string const& name, int size,
-        clock_type::rep expiration_seconds, clock_type& clock, beast::Journal journal,
+        clock_type::duration expiration, clock_type& clock, beast::Journal journal,
```

Comment:
Suggestion: change all instances in this file of `clock_type::duration` -> `std::chrono::seconds`

Reply:
This would subtly change behavior because of this line: https://github.com/HowardHinnant/rippled/blob/20e58c96fb259e4b5985ff49bfbb4d5c4462e821/src/ripple/basics/TaggedCache.h#L189

Currently `clock_type::duration` is `nanoseconds`, exported from `chrono::steady_clock`.  And the computation of `when_expire` on L189 is capable of producing fractional `seconds` even when `m_target_age` starts off with integral seconds.  And now that I look closer, the same is also true on L185.

Since so much of rippled runs on a 1-second-precision heartbeat, this change probably wouldn't cause big problems.  But I'm seeing only risk and no benefit to making it.